jenkins-client-it-docker/src/test/java/com/offbytwo/jenkins/integration/NoExecutorStartedPluginManagerIT.java

```diff
@@ -33,12 +33,12 @@ public void getPluginsShouldReturn9ForJenkins20() {
     }
 
     @Test
-    public void getPluginsShouldReturn27ForJenkins1651() {
+    public void getPluginsShouldReturn28ForJenkins1651() {
         JenkinsVersion jv = jenkinsServer.getVersion();
         if (jv.isLessThan("1.651") && jv.isGreaterThan("1.651.3")) {
```

Comment:
Ah cool, I don't think that will ever evaluate to false, so might be worth updating.

Might be worth updating the function name too? So it doesn't need to be updated if any other plugins are added (or goes out of sync with the actual amount)? wdyt?